diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -52,8 +52,7 @@
 include(GetGitRevisionDescription)
 include(InstallRequiredSystemLibraries)
 include(FindPkgConfig)
-include(ExternalProject)
-include(BuildLibMPEG2)
+PKG_SEARCH_MODULE(MPEG2 REQUIRED libmpeg2)

 use_cxx11( )

@@ -62,4 +61,3 @@
-build_libmpeg2( )

 message(STATUS "Target: ${CMAKE_SYSTEM_NAME} ${CMAKE_TARGET_ARCHITECTURES}")

@@ -113,7 +111,6 @@
 add_subdirectory(video)
 add_subdirectory(vldp)

-add_dependencies( vldp libmpeg2 )
 add_dependencies( ldp-out vldp )
 add_dependencies( game vldp )
 add_dependencies( sound vldp )
@@ -160,3 +157,5 @@
 set(CPACK_PACKAGE_VERSION_PATCH "${VER_PATCH}")

 include(CPack)
+install(TARGETS hypseus DESTINATION bin)
+install(DIRECTORY ../pics ../fonts ../sound DESTINATION /usr/share/daphne)
diff --git a/src/manymouse/CMakeLists.txt b/src/manymouse/CMakeLists.txt
--- a/src/manymouse/CMakeLists.txt
+++ b/src/manymouse/CMakeLists.txt
@@ -9,5 +9,6 @@
 )

 add_library( manymouse ${LIB_SOURCES} ${LIB_HEADERS} )
+target_compile_definitions( manymouse PRIVATE SUPPORT_XINPUT2=0 )
 target_link_libraries( manymouse ${CMAKE_DL_LIBS} )

diff --git a/src/vldp/vldp.h b/src/vldp/vldp.h
--- a/src/vldp/vldp.h
+++ b/src/vldp/vldp.h
@@ -36,2 +36,4 @@
 #include <SDL.h> // only used for threading
+
+#include <mpeg2dec/mpeg2.h>

diff --git a/src/vldp/vldp_internal.h b/src/vldp/vldp_internal.h
--- a/src/vldp/vldp_internal.h
+++ b/src/vldp/vldp_internal.h
@@ -27,7 +27,6 @@

 #include "vldp.h" // for the VLDP_BOOL definition and SDL.h

-#include <mpeg2.h>

 // this is which version of the .dat file format we are using
 #define DAT_VERSION 3
