--- a/es-app/src/guis/GuiNetPlay.cpp	2024-07-26 16:05:45.506880700 +0900
+++ b/es-app/src/guis/GuiNetPlay.cpp	2024-07-26 16:06:58.986846282 +0900
@@ -45,6 +45,7 @@ static std::map<std::string, std::string
 	{ "FB Alpha 2012", "fbalpha2012" },
 	{ "FB Alpha", "fbalpha" },
 	{ "FinalBurn Neo", "fbneo" },
+	{ "FinalBurn Neo DSNO", "fbneo-dsno" },
 	{ "FCEUmm", "fceumm" },
 	{ "Flycast", "flycast" },
 	{ "FreeIntv", "freeintv" },
