diff --git a/vita3k/modules/SceCommonDialog/SceCommonDialog.cpp b/vita3k/modules/SceCommonDialog/SceCommonDialog.cpp
index d31f70bc..06827b4b 100644
--- a/vita3k/modules/SceCommonDialog/SceCommonDialog.cpp
+++ b/vita3k/modules/SceCommonDialog/SceCommonDialog.cpp
@@ -1306,6 +1306,7 @@ EXPORT(int, sceSaveDataDialogInit, const SceSaveDataDialogParam *p) {
     SceSaveDataDialogUserMessageParam *user_message;
     SceSaveDataDialogSystemMessageParam *sys_message;
     SceSaveDataDialogProgressBarParam *progress_bar;
+    SceSaveDataDialogErrorCodeParam *error_code;
     std::vector<SceAppUtilSaveDataSlot> slot_list;
 
     emuenv.common_dialog.savedata = {};
@@ -1408,6 +1409,29 @@ EXPORT(int, sceSaveDataDialogInit, const SceSaveDataDialogParam *p) {
             }
         }
         break;
+    case SCE_SAVEDATA_DIALOG_MODE_ERROR_CODE:
+        error_code = p->errorCodeParam.get(emuenv.mem);
+        initialize_savedata_vectors(emuenv, 1);
+        emuenv.common_dialog.savedata.slot_id[0] = error_code->targetSlot.id;
+        emuenv.common_dialog.savedata.mode_to_display = SCE_SAVEDATA_DIALOG_MODE_FIXED;
+        emuenv.common_dialog.savedata.list_empty_param[0] = error_code->targetSlot.emptyParam.get(emuenv.mem);
+        check_save_file(0, emuenv, export_name);
+
+        emuenv.common_dialog.savedata.btn_num = 1;
+        emuenv.common_dialog.savedata.btn[0] = lang::get(lang::str::ok);
+        emuenv.common_dialog.savedata.btn_val[0] = SCE_SAVEDATA_DIALOG_BUTTON_ID_OK;
+        switch (p->dispType) {
+        case SCE_SAVEDATA_DIALOG_TYPE_SAVE:
+            emuenv.common_dialog.savedata.msg = fmt::format("{}\n({})", lang::get(lang::str::could_not_save), log_hex(error_code->errorCode));
+            break;
+        case SCE_SAVEDATA_DIALOG_TYPE_LOAD:
+            emuenv.common_dialog.savedata.msg = fmt::format("{}\n({})", lang::get(lang::str::could_not_load), log_hex(error_code->errorCode));
+            break;
+        case SCE_SAVEDATA_DIALOG_TYPE_DELETE:
+            emuenv.common_dialog.savedata.msg = fmt::format("An error has occurred while deleting.\n({})", log_hex(error_code->errorCode));
+            break;
+        }
+        break;
     default:
         LOG_ERROR("Attempt to initialize savedata dialog with unknown mode: {}", log_hex(p->mode));
         break;
