--- a/pcsx2-qt/GameList/GameListWidget.cpp
+++ b/pcsx2-qt/GameList/GameListWidget.cpp
@@ -75,21 +75,36 @@
 
 	void setFilterType(GameList::EntryType type)
 	{
+#if QT_VERSION >= QT_VERSION_CHECK(6, 10, 0)
 		beginFilterChange();
 		m_filter_type = type;
 		endFilterChange(Direction::Rows);
+#else
+		m_filter_type = type;
+		invalidateFilter();
+#endif
 	}
 	void setFilterRegion(GameList::Region region)
 	{
+#if QT_VERSION >= QT_VERSION_CHECK(6, 10, 0)
 		beginFilterChange();
 		m_filter_region = region;
 		endFilterChange(Direction::Rows);
+#else
+		m_filter_region = region;
+		invalidateFilter();
+#endif
 	}
 	void setFilterName(const QString& name)
 	{
+#if QT_VERSION >= QT_VERSION_CHECK(6, 10, 0)
 		beginFilterChange();
 		m_filter_name = name;
 		endFilterChange(Direction::Rows);
+#else
+		m_filter_name = name;
+		invalidateFilter();
+#endif
 	}
 
 	bool filterAcceptsRow(int source_row, const QModelIndex& source_parent) const override
