--- a/Utilities/JITLLVM.cpp
+++ b/Utilities/JITLLVM.cpp
@@ -632,6 +632,8 @@ std::string jit_compiler::triple1()
 {
 #if defined(_WIN32)
 	return llvm::Triple::normalize(llvm::sys::getProcessTriple());
+#elif defined(ARCH_ARM64)
+	return llvm::Triple::normalize("aarch64-unknown-linux-gnu");
 #elif defined(__APPLE__) && defined(ARCH_X64)
 	return llvm::Triple::normalize("x86_64-unknown-linux-gnu");
 #elif (defined(__ANDROID__) || defined(__APPLE__)) && defined(ARCH_ARM64)
@@ -649,6 +651,8 @@ std::string jit_compiler::triple2()
 {
 #if defined(_WIN32) && defined(ARCH_X64)
 	return llvm::Triple::normalize("x86_64-unknown-linux-gnu");
+#elif defined(ARCH_ARM64)
+	return llvm::Triple::normalize("aarch64-unknown-linux-gnu");
 #elif defined(_WIN32) && defined(ARCH_ARM64)
 	return llvm::Triple::normalize("aarch64-unknown-linux-gnu");
 #elif defined(__APPLE__) && defined(ARCH_X64)
